src/cockroach_db/mod.rs

```diff
@@ -0,0 +1,127 @@
+use std::collections::BTreeMap;
+
+use testcontainers::{core::WaitFor, Image, ImageArgs};
+
+const DEFAULT_IMAGE_NAME: &str = "cockroachdb/cockroach";
+const DEFAULT_IMAGE_TAG: &str = "latest-v23.2";
```

Comment:
I'd like to have stable tags (best effort) as a default for reproducibility reasons using this module.

My suggestion would be "v23.2.3" for now.